mifosng-provider/src/integrationTest/java/org/mifosplatform/integrationtests/common/ClientChargesTest.java

```diff
@@ -0,0 +1,121 @@
+package org.mifosplatform.integrationtests.common;
```

Comment:
Licences need to be applied , refer https://github.com/openMF/mifosx/blob/develop/Contributing.md point 10